src/components/GalleryCreate/index.tsx

```diff
@@ -0,0 +1,227 @@
+import React, { useState, useEffect } from 'react';
+// import { store } from 'apis/firebase';
+import styled from 'styled-components';
+import ReactModal from 'react-modal';
+import closeButton from '../../assets/icons/closeButton.svg';
+
+function GalleryRead() {
+  const [modalOpen, setModalOpen] = useState(false);
+
+  const [imgFile, setImgFile] = useState<File | null>();
+  const [preview, setPreview] = useState<string | null>('');
+
+  const openModal = () => {
+    setModalOpen(true);
+  };
+
+  const closeModal = () => {
+    setModalOpen(false);
+    setPreview(null);
+  };
+
+  const onChangeImg = (event: React.ChangeEvent<HTMLInputElement>) => {
+    if (event.target.files !== null) {
+      const file = event.target.files[0];
+      setImgFile(file);
+    } else {
+      setImgFile(null);
+    }
+  };
+
+  useEffect(() => {
+    if (imgFile) {
+      const reader = new FileReader();
+      reader.onloadend = () => {
+        setPreview(reader.result as string);
+      };
+      reader.readAsDataURL(imgFile);
+    } else {
+      setPreview(null);
+    }
+  }, [imgFile]);
+
+  const imgRegistration = () => {
+    if (preview === null) {
+      alert('사진 업로드 후 등록해주세요');
+    } else {
+      closeModal();
+    }
+  };
+
+  // const imgRead = () => {
+  //   console.log('imgRead');
+  // };
+
+  return (
+    <>
+      <StyledGalleryContainer>
+        <StyledButton onClick={openModal}>+ 사진 등록하기</StyledButton>
+        {/* <ExImg src={imgList as string}></ExImg> */}
+      </StyledGalleryContainer>
+      <ReactModal isOpen={modalOpen} style={StyledModal}>
+        <StyledCloseImg src={closeButton} onClick={closeModal}></StyledCloseImg>
+        <StyledContainer>
+          <StyledImgContainer>
+            {preview === null ? (
```

Comment:
null은 falsy값이어서 아래와 같이 표현해보면 좋을 것 같습니다!
```suggestion
            {preview ? (
```

Reply:
preview를 <string | ArrayBuffer | null> 타입으로 선언해서 null 값을 따로 지정해줘야 적용이 되는거 같습니다..! null 값을 빼주면 적용이 안되더라구요.. 🥲

Reply:
엇 그래요?! 저는 잘 적용이 되는데 혹시 preview 뒤에 두 식을 안바꿔주셨나요!?
preview ? 로만 쓰면 null일 때는 false로 들어가서 

` {preview ? (
              <StyledUploadImg src={preview as string}></StyledUploadImg>
            ) : (
              <StyledImgText>⚠️ 파일을 업로드 해주세요</StyledImgText>
            )}`

이런 식으로 작성해야 동작해요!

<img width="535" alt="스크린샷 2023-09-15 105615" src="https://github.com/energizer-develop/Y_FE_WIKI/assets/81469686/0f9a8783-71cd-4190-b725-ce748bc1c8c5">
<img width="467" alt="스크린샷 2023-09-15 105625" src="https://github.com/energizer-develop/Y_FE_WIKI/assets/81469686/656ab14b-f862-49d2-a654-750186f13da0">

Reply:
아!! 단순히 null만 제거해줬네요.. 😅😅
잘 반영되네요!